cfpb/consumerfinance.gov
# Remove `form.less`

The CSS in `form.less` was added in https://github.com/cfpb/consumerfinance.gov/pull/4456 for the conference registration form, which was removed in https://github.com/cfpb/consumerfinance.gov/pull/7082

## Removals

- Remove `form.less`

## How to test this PR

1. PR checks should pass. You can check a page like http://localhost:8000/data-research/prepaid-accounts/search-agreements/ that has a `o-form_fieldset` class and see that the removal of the unordered list bullets in the filters is redundant and unneeded.
